configure.ac

```diff
@@ -301,6 +301,7 @@ AC_CHECK_FUNCS( futimes )
 AC_CHECK_FUNCS( wcslcpy lrand48_r killpg )
 AC_CHECK_FUNCS( backtrace_symbols getifaddrs )
 AC_CHECK_FUNCS( futimens clock_gettime )
+AC_CHECK_FUNCS( localtime_r )
```

Comment:
@zanchey: Do we need to support platforms which don’t define `localtime_r`? I’m really not wild about using `localtime`, since it’s not thread-safe. Would it be acceptable for `configure` to just halt with an error if `localtime_r` is undefined?